ballerina/Client.bal

```diff
@@ -96,34 +96,34 @@ public isolated client class Client {
         return nativeBatchExecute(self, sqlQueries);
     }
 
-    # Executes a SQL query, which calls a stored procedure. This can return results or not.
+    # Executes an SQL query, which calls a stored procedure. This may or may not return results.
     #
     # + sqlQuery - The SQL query
-    # + rowTypes - The array `typedesc` of the records to which the results needs to be returned
+    # + rowTypes - `typedesc` array of the records to which the results need to be returned
     # + return - Summary of the execution and results are returned in an `sql:ProcedureCallResult`, or an `sql:Error`
     remote isolated function call(sql:ParameterizedCallQuery sqlQuery, typedesc<record {}>[] rowTypes = [])
     returns sql:ProcedureCallResult|sql:Error = @java:Method {
         'class: "io.ballerina.stdlib.mssql.nativeimpl.CallProcessorUtils",
         name: "nativeCall"
     } external;
 
-    # Closes the SQL client and shuts down the connection pool.
+    # Closes the MSSQL client and shuts down the connection pool.
     #
-    # + return - Possible error when closing the client
+    # + return - Possible `sql:Error` when closing the client
```

Comment:
```suggestion
    # + return - An `sql:Error` when closing the client
```
Please see the guidelines on how to add the return statements.

Reply:
Fixed in 889718f